_layouts/aggregator.html

```diff
@@ -54,7 +54,7 @@ <h2>Machine translation</h2>
       {% if page.supported_apis.size == 1 %}
         <strong>1</strong> machine translation API supports {{ page.title }}.
       {% else %}
-        <strong>{{ page.supported_apis | size }}</strong> machine translation APIs support {{ page.title }}.
+        <strong>{{ page.supported_apis | size }}</strong> machine translation APIs are supported by the {{ page.title }} aggregator.
```

Comment:
To honor the "no passive voice" instruction, could we change it to... ?

`The {{ page.title }} aggregator supports <strong>{{ page.supported_apis | size }}</strong> machine translation APIs.`

Does it sound more natural with the passive voice?

Reply:
I think my idea was to have the bold part in front.  But I can change that.

Reply:
I think the prohibition on passive voice was more in sentences without "by".  e.g. "SMT is considered the best approach to machine translation."  By whom, exactly?